### PR TITLE
added views for profile roles and IE reviews

### PR DESCRIPTION
**Pull request by Sivam2313**

### CODE REVIEW

File: website/interview_exp/api/views.py [hunk 1]
```diff
@@ -47,13 +47,15 @@ class IEListView(ListCreateAPIView):
     ordering_fields = ['updated_at', 'total_Compensation', 'year']
 
     def get_queryset(self):
+        print('yes')
```
Comment (moinak878) [inline website/interview_exp/api/views.py]: remove this print statement.
`good to have : try using logging package and use logger.info and logger.error whenever needed `

File: website/interview_exp/api/views.py [hunk 2]
```diff
@@ -47,13 +47,15 @@ class IEListView(ListCreateAPIView):
...
         qs = Experiences.objects.all()
         user = self.request.user
         if user.profile.role == '3':
             return qs.filter(Q(user=user) | Q(verification_Status='Approved'))
         return qs
 
     def perform_create(self, serializer):
+        print("yes")
```
Comment (moinak878) [inline website/interview_exp/api/views.py]: remove

File: website/interview_exp/api/views.py [hunk 3]
```diff
@@ -181,7 +183,7 @@ def perform_update(self, serializer):
 class RevisionsListView(ListAPIView):
     serializer_class = RevisionSerializer
 
-    permission_classes = (IsMemberOrAbove,)
+    # permission_classes = (IsMemberOrAbove,)
```
Comment (moinak878) [inline website/interview_exp/api/views.py]: why is permission_classes commented out ?

File: website/interview_exp/api/views.py [hunk 4]
```diff
@@ -190,18 +192,20 @@ def get_queryset(self):
...
     serializer_class = RevisionSerializer
-    permission_classes = (IsMemberOrAbove,)
-    lookup_field = 'id'
+    # permission_classes = (IsMemberOrAbove,)
+    lookup_field = 'experience_id'
     lookup_url_kwarg = 'id'
 
     def get_queryset(self):
         qs = Revisions.objects.prefetch_related('experience').prefetch_related('reviewer').all()
+        if (qs[0].experience.verification_Status=="Approved"):
```
Comment (moinak878) [inline website/interview_exp/api/views.py]: good to have : try to use .first , .last etc wherever possible to avoid [0] (indexed based retrieval)

**Review by moinak878 — CHANGES_REQUESTED** [review]
@Sivam2313 go through the review comments and make required changes. will merge after that

File: website/interview_exp/api/views.py (hunk 3 above)
Comment (Sivam2313) [reply website/interview_exp/api/views.py]: This one was done by mistake fixed it now.

File: website/interview_exp/api/views.py (hunk 1 above)
Comment (Sivam2313) [reply website/interview_exp/api/views.py]: Removed print for now will add logger later

**Review by moinak878 — APPROVED** [review]
gtg

**Pull request merged**